frontend/src/components/HospitalMap.tsx

```diff
@@ -17,7 +17,27 @@ class HospitalMap extends React.Component {
     lng: -0.09,
     zoom: 13,
   }
-
+  componentDidMount() {
+    /**
```

Comment:
If you want to put a comment inside a function, please don't use the "documentation comment" convention (`/**`). It's confusing. Use `//` or `/*` instead.

Reply:
ok got it